packages/experiments/src/components/KeytipLayer/examples/KeytipLayer.Basic.Example.tsx

```diff
@@ -0,0 +1,237 @@
+  showModal: boolean;
+  showMessageBar: boolean;
+}
+
+export interface IKeytipMap {
+  [componentKeytipId: string]: IKeytipProps;
+}
+
+export class KeytipLayerBasicExample extends React.Component<{}, IKeytipLayerBasicExampleState> {
+
+  private startingKeySequence: IKeytipTransitionKey = { key: 'Meta', modifierKeys: [KeytipTransitionModifier.alt] };
+  private keytipMap: IKeytipMap = {};
+
+  constructor(props: {}) {
+    super(props);
+
+    // Setup state
+    this.state = {
+      showModal: false,
+      showMessageBar: false
+    };
+
+    // Setup keytips
+    this.keytipMap.Pivot1Keytip = {
+      content: 'A',
+      keySequences: [{ keys: ['a'] }]
+    } as IKeytipProps;
+
+    this.keytipMap.Pivot2Keytip = {
+      content: 'B',
+      keySequences: [{ keys: ['b'] }]
+    } as IKeytipProps;
+
+    this.keytipMap.Button1Pivot1Keytip = {
+      content: '1B',
+      keySequences: addKeytipSequence(this.keytipMap.Pivot1Keytip.keySequences, { keys: ['1', 'b'] }),
+      onExecute: (el: HTMLElement) => {
+        el.click();
+      }
+    } as IKeytipProps;
+
+    this.keytipMap.Button2Pivot1Keytip = {
+      content: '1A',
+      keySequences: addKeytipSequence(this.keytipMap.Pivot1Keytip.keySequences, { keys: ['1', 'a'] }),
+      onExecute: (el: HTMLElement) => {
+        el.click();
+      }
+    } as IKeytipProps;
+
+    this.keytipMap.Button3Pivot1Keytip = {
+      content: 'M',
+      keySequences: addKeytipSequence(this.keytipMap.Pivot1Keytip.keySequences, { keys: ['m'] }),
+      onExecute: (el: HTMLElement) => {
+        el.click();
+      }
+    } as IKeytipProps;
+
+    this.keytipMap.CommandButton1Pivot2Keytip = {
+      content: '2',
+      keySequences: addKeytipSequence(this.keytipMap.Pivot2Keytip.keySequences, { keys: ['2'] }),
+      onExecute: (el: HTMLElement) => {
+        el.click();
+      }
+    } as IKeytipProps;
+
+    this.keytipMap.CommandButton2Pivot2Keytip = {
+      content: 'Y',
+      keySequences: addKeytipSequence(this.keytipMap.Pivot2Keytip.keySequences, { keys: ['y'] }),
+      onExecute: (el: HTMLElement) => {
+        el.click();
+      }
+    } as IKeytipProps;
+
+    this.keytipMap.CommandButton3Pivot2Keytip = {
+      content: 'LK',
+      keySequences: addKeytipSequence(this.keytipMap.Pivot2Keytip.keySequences, { keys: ['l', 'k'] }),
+      onExecute: (el: HTMLElement) => {
+        el.click();
+      }
+    } as IKeytipProps;
+  }
+
+  /* tslint:disable:jsx-ban-props jsx-no-lambda */
+  public render(): JSX.Element {
+    let divStyle = {
+      width: '50%',
+      display: 'inline-block',
+      verticalAlign: 'top'
+    };
+
+    return (
+      <div>
+        <p>Press Alt-Win to enable keytips, Esc to return up a level, and Alt-Win to exit keytip mode</p>
+        <div>
+          <div style={ divStyle }>
+            <div>
+              <ActionButton
+                data-ktp-id={ convertSequencesToKeytipID(this.keytipMap.Pivot1Keytip.keySequences) }
```

Comment:
feedback (not required to change in this PR)

abstract in a helper function so that caller doesn't know about data-*.

Also this is quite verbose. There needs to be a better experience than to call an API `convertSequencesToKeytipID` here for the caller.

```
private _keytips = this.autoDispose(registerKeytips({
  pivot1: { options },
  pivot2: { options }
}));

// ...and then:

<ActionButton
   ...this._keytips.pivot1
/>
```

Reply:
Right, this is only because the keytipProps haven't been added to ActionButton. We have to mock it right now because we haven't moved to office-ui-fabric-react. Hardcoding the data-ktp-id would be replaced with just defining keytipProps